tests/func-tests/observability_controller_test.go

```diff
@@ -19,18 +17,6 @@ const testName = "observability_controller"
 
 var _ = Describe("Observability Controller", Label(tests.OpenshiftLabel, testName), func() {
 	Context("PodDisruptionBudgetAtLimit", func() {
-		BeforeEach(func(ctx context.Context) {
-			cli := tests.GetControllerRuntimeClient()
-			tests.FailIfNotOpenShift(ctx, cli, testName)
```

Comment:
We do need these two lines. Even if it was mistakenly dropped in the original PR in main branch, we need to add it back there.